JAICore/jaicore-search/src/test/java/jaicore/search/algorithms/standard/bestfirst/RandomCompletionNodeEvaluatorTester.java

```diff
@@ -160,40 +161,40 @@ public void cancel() {
 				timeoutForNodeEvaluationInMs);
 	}
 
-	public RandomCompletionBasedNodeEvaluator<QueenNode, Double> getSeededNodeEvaluator(int seed, int numSamples) {
-		return getNodeEvaluator(n -> Math.random(), seed, numSamples, -1);
+	public RandomCompletionBasedNodeEvaluator<QueenNode, Double> getSeededNodeEvaluator(final int seed, final int numSamples) {
+		return this.getNodeEvaluator(n -> Math.random(), seed, numSamples, -1);
 	}
 
 	@Override
 	public RandomCompletionBasedNodeEvaluator<QueenNode, Double> getNodeEvaluator() {
-		return getSeededNodeEvaluator(0, 3);
+		return this.getSeededNodeEvaluator(0, 3);
 	}
 
 	@Override
 	public RandomCompletionBasedNodeEvaluator<QueenNode, Double> getBusyNodeEvaluator() {
-		return getTimedNodeEvaluator(-1);
+		return this.getTimedNodeEvaluator(-1);
 	}
 
 	@Override
-	public RandomCompletionBasedNodeEvaluator<QueenNode, Double> getTimedNodeEvaluator(int timeoutInMS) {
-		return getNodeEvaluator(new BusyObjectEvaluator<>(), 0, 3, timeoutInMS);
+	public RandomCompletionBasedNodeEvaluator<QueenNode, Double> getTimedNodeEvaluator(final int timeoutInMS) {
+		return this.getNodeEvaluator(new BusyObjectEvaluator<>(), 0, 3, timeoutInMS);
 	}
 
 	@Override
 	public Collection<Node<QueenNode, Double>> getNodesToTest(
-			RandomCompletionBasedNodeEvaluator<QueenNode, Double> ne) {
-		StandardBestFirst<QueenNode, String, Double> bf = getBF(new AlternativeNodeEvaluator<>(n -> 0.0, ne)); // the n
-																												// ->
-																												// 0.0
-																												// is
-																												// not
-																												// really
-																												// used
-																												// except
-																												// for
-																												// efficient
-																												// initialiazion
-//		bf.setLoggerName("testedalgorithm");
+			final RandomCompletionBasedNodeEvaluator<QueenNode, Double> ne) {
+		StandardBestFirst<QueenNode, String, Double> bf = this.getBF(new AlternativeNodeEvaluator<>(n -> 0.0, ne)); // the n
+		// ->
```

Comment:
maybe we can reduce this comment to a single line or two

Reply:
yep